packages/nulp_elite/src/components/header.js

```diff
@@ -640,7 +640,7 @@ function Header({ globalSearchQuery }) {
               {/* Check if roles array is empty or contains "PUBLIC" */}
 
               {/* {accessWorkspace && ( */}
-              {roleNames.some((role) => ["CONTENT_CREATOR"].includes(role)) && (
+              {roleNames.some((role) => ["CONTENT_CREATOR", "CONTENT_REVIEWER"].includes(role)) && (
```

Comment:
@suraj-tekdi also required org_admin too.

Have u confirmed for org_admin can able to access the WOrkspace?